prj.conf

```diff
@@ -4,104 +4,153 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+# This sample uses Kconfig.defaults to set options common for all
+# samples. This file should contain only options specific for this sample
+# or overrides of default values.
+
 # Enable CHIP
 CONFIG_CHIP=y
 CONFIG_CHIP_ENABLE_PAIRING_AUTOSTART=y
 CONFIG_CHIP_PROJECT_CONFIG="src/chip_project_config.h"
+# 32774 == 0x8006 (example lock-app)
```

Comment:
How about 0x8000 = 32768 ? It is not allocated to any device type https://github.com/project-chip/connectedhomeip/blob/master/docs/examples/discussion/PID_allocation_for_example_apps.md and it has corresponding DAC in the examples.

Reply:
done